### ISSUE TITLE
[1.7.1-beta.17] Uncaught TypeError on startup of cordova apps

### ISSUE THREAD

**Issue by rj-david** [meteor/meteor#10067]
Getting this error when starting up a cordova app

Error 1:
```
Uncaught TypeError: Cannot read property 'versions' of undefined
    at autoupdate_cordova.js (autoupdate.js:45)
    at fileEvaluate (modules-runtime.js:332)
    at Module.require (modules-runtime.js:234)
    at require (modules-runtime.js:254)
    at autoupdate.js:136
    at autoupdate.js:143
```

Error 2 (most probably due to Error 1:
```
Uncaught TypeError: Cannot read property 'Autoupdate' of undefined
    at global-imports.js:22
```

**Comment by benjamn** [comment]
This should be fixed in 1.7.1-beta.18. Thanks for reporting!

**Comment by rj-david** [comment]
Fix confirmed. Thanks
